src/lightning_app/cli/app-template/tests/test_placeholdername_app.py

```diff
@@ -10,7 +10,7 @@
 import os
 from contextlib import redirect_stdout
 
-from lightning.app.testing.testing import application_testing, LightningTestApp
+from lightning_app.testing.testing import application_testing, LightningTestApp
```

Comment:
Wondering how we were bel to test App as stand-alone when these imports shall be breaking it...